admin/api_access_tracking.go

```diff
@@ -177,18 +177,6 @@ func (a *AccessTrackingApiService) listAccessLogsByClusterNameExecute(r ListAcce
 	localVarHeaderParams := make(map[string]string)
 	localVarQueryParams := url.Values{}
 	localVarFormParams := url.Values{}
-	if strlen(r.groupId) < 24 {
```

Comment:
This type of validation was never part of any requirement or covered in spec. 
We inherited it through upstream. Errors that are reported aren't api errors - introducing unneeded complexity.